lib/exq/manager/server.ex

```diff
@@ -109,6 +109,21 @@ defmodule Exq.Manager.Server do
     Enqueuer.enqueue_in(state.enqueuer, from, queue, offset, worker, args)
     {:noreply, state, 10}
   end
+  
+  def handle_call({:subscribe, queue}, from, state) do
+    updated_state = add_queue(state, queue)
+    {:reply, :ok, updated_state}
```

Comment:
Ah, interesting. Is this because we need the handle_info method to be triggered because a timeout of 0?